scripts/security/certs-verify.sh

```diff
@@ -5,7 +5,7 @@ set -o nounset \
     -o verbose
 
 # See what is in each keystore and truststore
-for i in kafka1 kafka2 client schemaregistry restproxy connect connectorSA controlcenter ksqlDBserver ksqlDBUser appSA badapp clientListen zookeeper mds
+for i in kafka1 kafka2 client schemaregistry restproxy connect connectorSA control-center ksqldb-server ksqlDBUser appSA badapp clientListen zookeeper mds
```

Comment:
@javabrett Can you please elaborate why this PR adds dashes to `control-center ksqldb-server` but not, for example, `ksqlDBUser schemaregistry restproxy`?

Reply:
Only these two as a combination of a) these were the only two server certs I was working with in the context of getting the WebSockets working for HTTPS per this PR, and b) they are the only hosts I noticed where their Docker hostnames did not match their name in the certificates iterator, therefore they had non-matching CN/DNS SANs.

Note that this is unimportant in the context of this PR, since access is via port-mapped `localhost` Docker host, so as long as these certs have `localhost`, that part works fine, hence the name-fixes were originally in PR #278 but now included here.

Checking the others:
- [x] `restproxy` -> `hostname: restproxy`
- [x] `schemaregistry` -> inconsistent in `docker-compose.yml` as `hostname:` directive missing, but defaults to container name `schemaregistry`, so correct.
- [x] `ksqlDBUser` -> appears to be a client cert?  Name probably doesn't matter.

Now the real question ... since `control-center` has a hyphenated Docker hostname, why not `rest-proxy` and `schema-registry` 😸 ?

Reply:
@javabrett  to make sure we are in sync, should the desired end state be that they all have hyphens?  If you agree, I can file a separate jira to change RP and SR to `rest-proxy` and `schema-registry` everywhere